scripts/build-retention/monitoring/prometheus.yml

```diff
@@ -0,0 +1,29 @@
+global:
+  scrape_interval: 5s
+  external_labels:
+    monitor: 'disk-monitor'
+
+rule_files:
+# All mountpoints in alert.rules have been manually mounted onto the node.
+  - 'alert.rules'
+
+# Change all ips below to that of the mounted node.
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs: 
+      - targets: ['10.230.242.163:9090']
```

Comment:
Should not mention IP hardcoded in this file

Reply:
IP is reference to build-retention node where deployment is running & LCO NFS is mounted. Developer will change IP on deploying to next node. Current IP shows deployed node and should be treated as placeholder reference.